starfish/io.py

```diff
@@ -94,33 +92,46 @@ def set_aux(self, key, img):
     def max_proj(self, *dims):
         return self.image.max_proj(*dims)
 
-    def squeeze(self, bit_map_flag=False):
-        first_dim = self.image.num_hybs * self.image.num_chs * self.image.num_zlayers
+    def squeeze(self) -> np.ndarray:
+        """return an array that is linear over categorical dimensions and z
+
+        Returns
+        -------
+        np.ndarray :
+            array of shape (num_hybs + num_channels + num_z_layers, x, y).
```

Comment:
i think @dganguli  said we're doing hyb,ch,z,y,x.

or should be...?

Reply:
Yep, that's what the `ImageStack.numpy_array` object is/should be. This is a weird function that I hope will go away as a result of #226.